tools/docker/substrate-all-in-one/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[profile.release]
```

Comment:
@RafaelAPB @CatarinaPedreira Where is this file being used by the container? I could not find a reference to it in the `Dockerfile`

Reply:
This is actually not needed anymore, I'm removing it